test/util/display_SYSINFO_settings.py

```diff
@@ -1,31 +1,28 @@
-# Copyright 2016, EMC, Inc.
+'''
+Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
 
-"""
-  Purpose:
-  This script will generate a list of discovered nodes and display the system information for the nodes 
-"""
+Purpose:
+  This script will generate a list of discovered nodes and display the system information for the nodes
+'''
 
 import fit_path  # NOQA: unused import
-import os
-import sys
-import subprocess
-
 import fit_common
-import test_api_utils
+
+fit_common.VERBOSIY = 1
```

Comment:
just talked with George and these are intentional and are needed to _keep_ all the debug stuff out of here. If there are other changes to do, I would say add a comment along the lines of:
"# This a CLI utility and not a test. It is only run from a terminal, so make verbosity _not_ show debug information"

If there isn't any other changes, never mind :)